lib/reporters/checkstyle.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Author: Josh Hagins
+ * https://github.com/jawshooah
+ *
+ * Modified from the original by: Boy Baukema
+ * https://github.com/relaxnow
+ */
+
+module.exports = {
+  reporter: function(results) {
```

Comment:
There are [checkstyle](https://www.npmjs.com/package/jshint-checkstyle-reporter) [reporters](https://www.npmjs.com/package/jshint-jenkins-checkstyle-reporter) for [jshint](https://www.npmjs.com/package/jshint-checkstyle-file-reporter), but the errors emitted by `vnu.jar` are formatted differently, with differently named values.

A good solution might be to require one of these modules, pre-process the results from `vnu.jar`,  and feed them to the reporter.